sonic_platform_base/sonic_sfp/ext_media_handler_qsfp56_dd.py

```diff
@@ -0,0 +1,227 @@
+########################################################################
+# DellEMC
+#
+# Module contains the ext media drivers for QSFP56-DD modules
+#
+########################################################################
+
+from ext_media_utils import *
+from ext_media_handler_base import media_static_info
+
+QSFP56_DD_CMIS_1_LENGTH_ADDR = media_eeprom_address(offset=146)
```

Comment:
We have a QSFP-DD parser which support CMIS modules, is it possible to reuse it here?